### PR TITLE
history: list last n commands from the history

### PR DESCRIPTION
**Pull request by SirGlorg**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/history.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 `history`
 
+- Display last 20 commands from the history list:
+
+`history 20`
```
Comment (zdroid) [inline pages/common/history.md:11]:
```suggestion
- Display the last 20 commands:

`history {{20}}`
```

**Comment by einverne** [comment]
I have tested this command under Ubuntu, and I cannot make this `display the last 20 commands` work. Did I do something wrong?

**Comment by zdroid** [comment]
@einverne for me `history 20` works fine. Did you miss to remove `{{ }}`?

**Comment by SirGlorg** [comment]
@einverne https://www.gnu.org/software/bash/manual/html_node/Bash-History-Builtins.html documentation supports the `n` argument. I tested on macOs, also works. Can you check the `man history` or `history -h` and check, if Ubuntu's implementation happen to miss this option? I'd be surprised, but it does sometimes (like the `find` command, which I hate with passion for its varying behavior on different distributions).

**Comment by einverne** [comment]
Maybe some problems with zsh, I just tested on another server it works.

**Pull request merged**